launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -249,7 +249,7 @@ def run(self):
         # write the flags to disk
         run_results = Updater.overall_update_status(results)
         Updater._write_updates_status_flag_to_disk(run_results)
-        if run_results == UpdateStatus.UPDATES_OK:
+        if run_results in {UpdateStatus.UPDATES_OK, UpdateStatus.REBOOT_REQUIRED}:
```

Comment:
I would recommend adding a code comment here explaining the behavior.

Reply:
:heavy_check_mark: